desc/utils.py

```diff
@@ -682,3 +682,6 @@ def broadcast_tree(tree_in, tree_out, dtype=int):
     # invalid tree structure
     else:
         raise ValueError("trees must be nested lists of dicts")
+
+
+PRINT_WIDTH = 60
```

Comment:
Is this width is based on the bootstrap objective? We should include a comment in the code saying why we chose this hard-coded number

Reply:
Yes, current longest name is Bootstrap one. Actual value is 58 I guess but I like round numbers :D

Reply:
Yes, current longest name is Bootstrap one. Actual value is 58 I guess but I like round numbers :D

Reply:
Yes, current longest name is Bootstrap one. Actual value is 58 I guess but I like round numbers :D

Reply:
Yes, current longest name is Bootstrap one. Actual value is 58 I guess but I like round numbers :D